npm/ux/utils.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function findParentByClassName(
```

Comment:
You don't need this helper. Use <https://developer.mozilla.org/en-US/docs/Web/API/Element/closest> .  See resultsTable.tsx for examples, which uses it extensively.